flow360/component/flow360_params/params_base.py

```diff
@@ -1163,6 +1195,10 @@ def names(self) -> List[str]:
         """return names of all boundaries"""
         return [k for k, _ in self if k not in [COMMENTS, TYPE_TAG_STR]]
 
+    def items(self) -> List[str]:
+        """return names of all boundaries"""
+        return [obj for k, obj in self if k not in [COMMENTS, TYPE_TAG_STR]]
+
```

Comment:
with dicts, items should return `key, item` tuple, right, https://docs.python.org/3.12/library/stdtypes.html#dict.items

If you want to access item by name, there is `[]` accessor implemented.

Reply:
Good point. Removed .items() method.